superagi/models/resource.py

```diff
@@ -58,6 +58,11 @@ def validate_resource_type(storage_type):
 
         if storage_type not in valid_types:
             raise InvalidResourceType("Invalid resource type")
-
+    
+    @classmethod
+    def get_all_resources_from_run_ids(cls,session,execution_ids):
```

Comment:
find_resources_by_run_ids

rename execution_ids param to run_ids

Reply:
renamed